modules/layers/src/contour-layer/marching-squares.js

```diff
@@ -1,43 +1,51 @@
 // All utility mehtods needed to implement Marching Squres algorithm
 // Ref: https://en.wikipedia.org/wiki/Marching_squares
 import assert from 'assert';
```

Comment:
Just noticed this - `assert` is not a dependency and only works because of webpack polyfill. This has been replaced in the core module by https://github.com/uber/deck.gl/blob/master/modules/core/src/utils/assert.js and the layers module by https://github.com/uber/deck.gl/blob/master/modules/layers/src/geojson-layer/geojson.js#L24.

Generally speaking, we only use assert checks on things that involve user data or user implementation. If it is something strictly generated by the layer itself, you should implement a safe guard instead of throwing an error.

This needs to be fixed in a separate PR.

Reply:
Done, addressed in 2438